asciidoc/courses/workshop-graphrag-introduction/modules/3-querying/lessons/5-questions/lesson.adoc

```diff
@@ -1,5 +1,8 @@
 = Questions
 :type: conversation
 :order: 5
+:optional: true
```

Comment:
I dont think this should be optional, as per previous comment.

Making the last lesson of the last module optional will mean the learner will never see it.

GraphAcademy will immediately go to the "congratulations" screen once they have completed it.